Change NSDocumentDirectory to NSCachesDirectory

allanwolski opened the pull request:
I moved to the cache directory not to list files in the Files app.

allanwolski committed:
Merge pull request #1 from CrossPT/master

Syncing fork to the original repository

allanwolski committed:
Change to use NSCachesDirectory

amerchDev committed:
cleanup cache and fix temp file name current always results in .pdf as prefix

amerchDev committed:
cleanup cache and fix temp file name current always results in .pdf as prefix

amerchDev committed:
fixed spacing

amerchDev committed:
rxdart version

amerchDev committed:
add prefix so cache cleanup works better to have only 1 pdf cached at a time

amerchDev committed:
lower case check

RafaelR7 commented:
I hope this PR be approved as soon as possible. This PR might solves the problem of files been visible on IOS Files

allanwolski commented:
Any updates @CrossPT?

lohanidamodar committed:
updated for latest version of flutter

lohanidamodar committed:
and upgraded version of viewer using the pageview
#supports swipe gestures to move to next page and previous page
#loads page as required

lohanidamodar committed:
disable navigation buttons based on the _pageNumber value

lohanidamodar committed:
changing names

lohanidamodar committed:
link to original project

lohanidamodar committed:
updating viewer v2 as main viewer

lohanidamodar committed:
export show IndicatorPosition enum

lohanidamodar committed:
navigation builder

lohanidamodar committed:
hide picker on indicator tap if showPicker is false

lohanidamodar committed:
version 1.0.0

lohanidamodar committed:
name changing

lohanidamodar committed:
disable slide on zooming

lohanidamodar committed:
version 1.0.1

lohanidamodar committed:
upgraded plugins to latest version

lohanidamodar committed:
hide picker for one page document

lohanidamodar committed:
remove rxdart dependency

lohanidamodar committed:
upgraded to androidX

lohanidamodar committed:
allow passing optional headers for url documents

lohanidamodar committed:
formatting

lohanidamodar committed:
version 1.1.0

lohanidamodar committed:
option to enable/disable swipe navigation

lohanidamodar committed:
change scroll direction

lohanidamodar committed:
version 1.1.1

lohanidamodar committed:
format viewer

lohanidamodar committed:
pub publish workflow

lohanidamodar committed:
allow preloading all pages

lohanidamodar committed:
version 1.1.2

lohanidamodar committed:
Merge branch 'release/1.1.2'

lohanidamodar committed:
Merge tag '1.1.2' into develop

1.1.2

Commit:
Merge branch 'master' of https://github.com/lohanidamodar/pdf_viewer

Commit:
update podspec filename

lohanidamodar committed:
pub version in readme

lohanidamodar committed:
allow passing in controller

lohanidamodar committed:
changelog and pubspec version upgrade

lohanidamodar committed:
Merge branch 'feature/controller' into develop

lohanidamodar committed:
Merge branch 'release/1.1.3'

lohanidamodar committed:
Merge tag '1.1.3' into develop

1.1.3

lohanidamodar committed:
Merge pull request #1 from amerchDev/master

Cleanup temp files and fix iOS build

lohanidamodar committed:
version and changelog

lohanidamodar committed:
Merge branch 'release/1.1.4'

lohanidamodar committed:
Merge tag '1.1.4' into develop

1.1.4

lohanidamodar committed:
fixing initial page using page controller

lohanidamodar committed:
pubspec version upgrade and changelog

lohanidamodar committed:
Merge branch 'release/1.1.5'

lohanidamodar committed:
Merge tag '1.1.5' into develop

1.1.5

lohanidamodar committed:
expose zoom widget parameters

lohanidamodar committed:
version upgrade and changelog

lohanidamodar committed:
Merge branch 'release/1.1.6'

lohanidamodar committed:
Merge tag '1.1.6' into develop

1.1.6

lohanidamodar committed:
updated to work with flutter 1.20+

lohanidamodar committed:
Merge branch 'release/1.2.0'

lohanidamodar committed:
Merge tag '1.2.0' into develop

v1.2.0

lohanidamodar committed:
updated pubspec to match latest updates

lohanidamodar committed:
Merge branch 'release/1.2.0'

allanwolski committed:
Merge remote-tracking branch 'pdf_viewer/master'
